app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -338,8 +337,11 @@ def find_gpg_key
     end
 
     def repository_params
-      keys = [:url, :gpg_key_id, :unprotected, :name, :checksum_type, :docker_upstream_name, :download_policy, :mirror_on_sync, :ostree_branches => []]
+      keys = [:download_policy, :mirror_on_sync, :ostree_branches => []]
```

Comment:
Guessing that ostree_branches parameter might clash with some of the ostree update work